CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 3.0.x]
 ### Added
+- Enabled Async Shard Batch Fetch by default ([#18139](https://github.com/opensearch-project/OpenSearch/pull/18139))
```

Comment:
This will need to go in the release notes if we merge this into the 3.0 release. Also the changelog entry on main should be removed if this ends up being released in 3.0.